dataactbroker/handlers/fileHandler.py

```diff
@@ -151,18 +151,13 @@ def getErrorReportURLsForSubmission(self, is_warning = False):
             # Unexpected exception, this is a 500 server error
             return JsonResponse.error(e, StatusCode.INTERNAL_ERROR)
 
-    def get_signed_url_for_submission_file(self):
+    def get_signed_url_for_submission_file(self, submission):
```

Comment:
We've got DB-1538 in the sprint now to address this; salright to leave this until then? We'll need the submission for the file verification.